plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/ImplementSqlServerStddevPop.java

```diff
@@ -44,18 +46,19 @@
     public Pattern<AggregateFunction> getPattern()
     {
         return basicAggregation()
-                .with(functionName().equalTo("stddev_pop"))
+                .with(functionName().matching(name -> name.equalsIgnoreCase("stddev_pop")))
```

Comment:
i see `.with(functionName().equalto` 10 times in the codebase.

We should rather change the `functionName().equalto` impl, that "workaround" it in every place

Reply:
I'm backing away all changes except the comment explaining why covar/covar_pop are missing.

Will make suggested changes in a follow-up PR before the coming release.